scripts/apps/search/views/search-filters.html

```diff
@@ -63,7 +63,7 @@
         </ul>
     </div>
 
-    <div sd-toggle-box data-style="circle" data-title="{{:: 'Services' | translate }}" ng-hide="isEmpty('category')" data-open="true">
+    <div sd-toggle-box data-style="circle"  data-title="{{:: category.display_name | translate }}" ng-hide="isEmpty('category')" data-open="true">
```

Comment:
Don't use `translate` on a dynamic value. See where that display name is getting set and translate there.